### ISSUE TITLE
AttributeError: 'list' object has no attribute 'split'

### ISSUE THREAD

**Issue by infinitnet** [RocketChat/Rocket.Chat.Ansible#4]
Trying on a fresh CentOS 7 system, but no luck so far. `upgrade.yml` is missing from this repo but included via main.yml and after removing the `include` of that missing YAML I get this:

```
TASK: [RocketChat.Server | Ensure MongoDB Server is present] ****************** 
failed: [chat.test.com] => {"failed": true, "parsed": false}
Traceback (most recent call last):
  File "/root/.ansible/tmp/ansible-tmp-1452005024.71-58413315786860/yum", line 3371, in <module>
    main()
  File "/root/.ansible/tmp/ansible-tmp-1452005024.71-58413315786860/yum", line 983, in main
    disablerepo, disable_gpg_check)
  File "/root/.ansible/tmp/ansible-tmp-1452005024.71-58413315786860/yum", line 867, in ensure
    items = pkgspec.split(',')
AttributeError: 'list' object has no attribute 'split'
OpenSSH_6.6.1, OpenSSL 1.0.1e-fips 11 Feb 2013
debug1: Reading configuration data /etc/ssh/ssh_config
debug1: /etc/ssh/ssh_config line 56: Applying options for *
debug1: auto-mux: Trying existing master
debug1: mux_client_request_session: master session id: 2
Shared connection to chat.test.com closed.


FATAL: all hosts have already failed -- aborting
```

**Comment by cmacrae** [comment]
Hey @infinitnet - as mentioned on #3: `upgrade.yml` is certainly not missing: https://github.com/RocketChat/Rocket.Chat.Ansible/blob/master/tasks/upgrade.yml

It is also not included by `main.yml` because the `notify` param on the task which informs the role an upgrade is needed is commented out. I don't see how a change in that task would notify the respective handler if the code is not actually executed.  

As mentioned in #3, I'll try to replicate this.

**Comment by infinitnet** [comment]
Should be related to #3, closing this one for now and will re-open it if it turns out otherwise.

**Comment by cmacrae** [comment]
Yep, got this fixed :+1: I'll mention this in #3
